MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs

```diff
@@ -177,15 +171,15 @@ public virtual void Show(IMvxIosView view, MvxViewModelRequest request)
                 }
             }
 
-            if (TabBarViewController != null && TabBarViewController.ShowChildView(viewController))
+            if (MasterNavigationController != null)
             {
+                PushViewControllerIntoStack(MasterNavigationController, viewController, attribute.Animated);
+
                 return;
             }
 
-            if (MasterNavigationController != null)
+            if (TabBarViewController != null && TabBarViewController.ShowChildView(viewController))
```

Comment:
@nmilcoff   mmm,  actually, i think we should remove the code `if (TabBarViewController != null && TabBarViewController.ShowChildView(viewController))
            {
                return;
            }` in `ShowChildViewController` method.  Because if user set `MvxTabPresentation` attribute for a viewcontroller, It will just call the `ShowTabViewController` method.